lib/get-unified-config.js

```diff
@@ -260,6 +261,11 @@ function getUnifiedConfig () {
     delete results.global.samplePercent;
   }
 
+  // remove this setting unless we're in the lambda environment
```

Comment:
rather than do this you probably want to set `eeid: 'lambda'` in the definition. that will cause it to omit the setting if the execution environment is not lambda (as the other lambda-only settings).

Reply:
That's what I tried first in 9be515468d9889c53114656bf76d7cd3676b9602, unfortunately if a setting has the `default` property set, it will stick around.

Reply:
invert the setting? make it non-blocking?

Reply:
obviously your call, but it's a bug and i thought i'd offer a workaround. eeid should not set a default.